chart/release/values.yaml

```diff
@@ -1,17 +1,3 @@
-nats:
-  cluster:
-    enabled: true
-    replicas: 3
-  exporter:
-    enabled: false
-  logging:
-    debug: false
-    trace: false
-
-etcd:
-  auth:
-    rbac:
-      enabled: false
-  replicaCount: 3
-
-mayastorLogLevel: info
+mayastorPools:
+  - node: "NODE_NAME"
+    device: "DEVICE"
```

Comment:
Do you think it's useful to have this @reitermarkus? I guess it's good to see an example, but if it will error since these are dummies.